src/main/java/com/conveyal/r5/profile/ProfileRequest.java

```diff
@@ -135,10 +135,9 @@
     public int suboptimalMinutes = 5;
 
     /**
-     * The maximum duration of any transit trip found by this search. Results used to be very sensitive to this
-     * when we were using mean travel times, now we use medians so we could set this to 2 hours.
+     * The maximum duration of any transit trip found by this search.
      */
-    public int maxTripDurationMinutes = 4 * 60;
+    public int maxTripDurationMinutes = 2 * 60;
```

Comment:
We should verify that as expected the results are no longer super sensitive to this value.